open-metadata-implementation/user-interfaces/access-services-user-interface/src/main/static/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "egeria-ui",
+  "version": "0.0.1",
+  "description": "A starting point for Polymer apps",
+  "author": "The Polymer Authors",
+  "license": "BSD-3-Clause",
```

Comment:
Why BSD-3 ? Is this the license for polymer itself? How does it affect derived apps?
If supplied by them, do we not pull it down automatically and not checkin
I assume instead it's a sample, modified for our needs - in which case polymer aren't the authors? (subject to license/attribution)

Reply:
Changed.
No! It is generated like this by the polymer starter kit.
The purpose of the PR is to have a build with polymer from maven.
That's why, I made it with default app polymer template app.

Reply:
I was questioning the license -- if indeed it is 'their' code that we've modified I think we would have to stick with their license, and also add the necessary license text into the egeria top level docs.

If it's only in master temporarily, perhaps we should raise an issue to track the deviance .. acting as a reminder before any release? Or we go back to BSD-3. Any comments @mandy-chessell ?